### PR TITLE
src: fix pricestf socket connection fail loop

### PR DESCRIPTION
**Pull request by idinium96**

### CODE REVIEW

File: src/classes/BotManager.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export = class BotManager {
 
         this.socket.on('connect', () => {
             log.debug('Connected to socket server');
-            this.socket.emit('authentication', process.env.PRICESTF_API_TOKEN);
+            this.socket.emit('authentication', process.env.PRICESTF_API_TOKEN || undefined);
```
Comment (Nicklason) [inline src/classes/BotManager.ts]: I guess I thought that it should be undefined anyway, but people probably just copy-paste the config and don't fill in the api key.

Comment (ZeusJunior) [reply src/classes/BotManager.ts]: I mean most just change the filename, lol

Comment (idinium96) [reply src/classes/BotManager.ts]: yeah.. but leaving it empty will cause an infinite loop of connecting and then disconnected to socket server. I'm not sure why.

Comment (Nicklason) [reply src/classes/BotManager.ts]: it is because the socket keeps getting disconnected, it will then reconnect because of the code below

**Pull request merged**